lib/index_group.rb

```diff
@@ -15,6 +15,9 @@ class IndexGroup
     def initialize(base_uri, name, schema, search_config)
       @base_uri = base_uri
       @client = Services::elasticsearch(hosts: base_uri)
+      # Index creation can take longer than other requests, so create a separate
+      # client with a longer timeout
+      @index_creation_client = Services::elasticsearch(hosts: base_uri, timeout: 10)
```

Comment:
Sorry, I missed your second question. I think we want to keep the default timeout low(ish) because the other types of requests should be fast. We'd want to know if they became much slower, even if the first indication was request timeouts.